pkg/validator/validators.go

```diff
@@ -360,22 +391,53 @@ func validateMRCProvider(mrc *configv1alpha2.MeshRootCertificate) error {
 	return nil
 }
 
-// checkForExistingActiveMRC returns true if there are active MRCs in addition to the specified MRC
+// mrcWithMatchingIntentExists returns true if there are active MRCs in addition to the specified MRC
```

Comment:
Can we update this comment to not include "Active"? This method seems to return if there are any MRCs with the same intent and is not limited to Active MRCs.